custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2279,6 +2279,16 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
         vm.getDataForStep(vm.step);
     });
 
+    vm.isOlderThan90Days = function () {
+        var selectedDate = dateHelperService.getSelectedDate();
+        var currentDate = new Date();
+        currentDate.setHours(0, 0, 0, 0);
+        var diffTime = currentDate - selectedDate;
+        var diffDays = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+
+        return diffDays > 90;
```

Comment:
hey, I think here good to check with Nitish what he wants to show in case where there is no image in for example current month(when we are under 90 days). Then do the code according to that.

For example.  no image check was fine. What you could do is to do the same no image check as you did previous.  then what message to display can depend on the month. or something like that. 
Because this current which does not match with the 90 days stuff because it could be possible to have images but here it passes as more than 90 days. in which can we would show this message even when we have images.

Is all that making any sense?